dev-scripts/test-cdc-lifecycle-e2e.py

```
@@ -0,0 +1,718 @@
+        state["srv"], state["f"] = None, None
+
+        result["status"] = "pass"
+        log(f"✅ LIFECYCLE PASS: {label}", G)
+    except Exception as e:
+        result["error"] = str(e)
+        log(f"❌ LIFECYCLE FAIL: {label}: {e}", R)
+    finally:
+        try:
+            stop_ncps(state.get("srv"), state.get("f"))
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)